src/interactivity/base/RemoteConsoleControl.cpp

```diff
@@ -32,18 +32,23 @@ template<typename T>
     packet.code = signalCode;
     packet.data = payload;
 
-    DWORD dwWritten = 0;
-    if (!WriteFile(pipe, &packet, sizeof(packet), &dwWritten, nullptr))
+    DWORD bytesWritten = 0;
+    if (!WriteFile(pipe, &packet, sizeof(packet), &bytesWritten, nullptr))
     {
-        return NTSTATUS_FROM_WIN32(::GetLastError());
+        NT_RETURN_NTSTATUS(static_cast<NTSTATUS>(NTSTATUS_FROM_WIN32(::GetLastError())));

```

Comment:
i... thought we moved to wil because there was a macro that made this less ugly, not a macro that made it **more** ugly 😉

Reply:
For the glory of error tracing! 🥲
It's really weird that the `static_cast` is necessary...

Reply:
the only plus side to this mess is that it is more debuggable with the wil tracelog. You're right... wil was supposed to solve this but then on further inspection... someone implemented like 4 of the functions and not the ones I needed (`NT_RETURN_NTSTATUS_IF` and `NT_RETURN_NTSTATUS_IF_WIN32_BOOL_FALSE` etc.)